tiatoolbox/tools/registration/wsi_registration.py

```diff
@@ -1656,6 +1667,8 @@ def read_rect(
             resolution=resolution,
             units=units,
         )
+        transformed_location: tuple[int, int]
```

Comment:
I think this should not be required considering output of `self.get_transformed_location` is already specified.

Reply:
Yes that's true, I'll remove these